src/aap_eda/wsapi/consumers.py

```diff
@@ -72,12 +72,31 @@ class Event(Enum):
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%f%z"
 
 
+class DefaultConsumer(AsyncWebsocketConsumer):
+    """Default consumer for websocket connections.
+
+    This is the consumer to handle all the unexpected paths, it will close the
+    connection with an error message.
+    """
+
+    async def connect(self):
+        await self.accept()
+        await self.send('{"error": "invalid path"}')
+        await self.close()
+
+
 class AnsibleRulebookConsumer(AsyncWebsocketConsumer):
     async def receive(self, text_data=None, bytes_data=None):
         data = json.loads(text_data)
         logger.debug(f"AnsibleRulebookConsumer received: {data}")
 
-        msg_type = MessageType(data.get("type"))
+        try:
+            msg_type = MessageType(data.get("type"))
+        except ValueError:
+            logger.error(f"Unsupported message type: {data}")
+            await self.send('{"error": "unsupported message type"}')
+            await self.close()
```

Comment:
@mkanoor tell me what do you think, I'm not sure if here we want to close the connection.